src/main/cpp/t10_unique.cpp

```diff
@@ -26,4 +26,28 @@ using namespace std;
 
 int t10_unique() {
 
-}
+    int N = 1 ;
+    bool  a = 0 ;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The scope of the variable 'a' can be reduced.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4732197)